src/macaron/dependency_analyzer/cyclonedx.py

```diff
@@ -259,46 +258,26 @@ def add_latest_version(
                 logger.error("Could not parse dependency version number: %s", error)
 
     @staticmethod
-    def merge_configs(
-        config_deps: list[Configuration], resolved_deps: dict[str, DependencyInfo]
-    ) -> list[Configuration]:
-        """Merge the resolved dependencies into the manual config dependencies.
-
-        Manual configuration entries are prioritized over the automatically resolved dependencies.
+    def to_configs(resolved_deps: dict[str, DependencyInfo]) -> list[Configuration]:
+        """Convert the resolved dependencies into the format used by the Analyzer.
 
         Parameters
         ----------
-        config_deps : list[Configuration]
-            Dependencies defined in the configuration file.
         resolved_deps : dict[str, DependencyInfo]
             The automatically resolved dependencies.
 
         Returns
         -------
         list[Configuration]
-            The result list contains the merged dependencies.
+            The dependencies list to be used by the Analyzer.
```

Comment:
```suggestion
            The dependency list to be used by the Analyzer.
```

Reply:
Fixed in https://github.com/oracle/macaron/pull/844/commits/d00d4827d3f0f998788f91d9eadbe24a488464ca